test/lib/completions/ssh.exp

```diff
@@ -76,7 +76,7 @@ foreach h [get_known_hosts] {
         }
     }
 }
-assert_complete $hosts "ssh $char" $test -ltrim-colon-completions -expect-cmd-minus "$char"
+assert_complete $hosts "ssh $char" $test -expect-cmd-minus "$char"
```

Comment:
I don't have any desire to understand what `-ltrim-colon-completions` option is actually doing, or for what purpose. But this test is the only use of this option, so unless this is really testing something meaningful, I think eliminating the option altogether may be the simplest option.